.github/workflows/deploy.yml

```diff
@@ -20,10 +20,15 @@ jobs:
         run: |
            ./build.sh
 
+      - name: Get current date
+        id: currentdatetime
+        run: echo "::set-output name=datetime::$(date +'%Y%m%d%H%M%S')"
+
       - name: configure aws credentials
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@v2
         with:
-          role-to-assume: arn:aws:iam::012986738649:role/GitHubPushToECR
+          role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_NUMBER }}:role/GitHubPushToECR
```

Comment:
${{ secrets.AWS_ACCOUNT_NUMBER }}

This requires an environment variable setting on the repository

Reply:
Environment variable is now set for production, test points at current development for now as the AWS Test environment has no Githubpushtoecr role